components/instalation/instalation.css

```diff
@@ -0,0 +1,43 @@
+@import url(/landing.css);
+@import url(/variable.css);
+
+body{
+    display: flex;
+    max-width: 100%;
+    flex-wrap: wrap;
+}
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.drawer_component{
```

Comment:
For readability, one space is suggested before the curly brace start in the entire file. Extension 'beautify' or 'prettier' might work well for this. Ctrl+Shift+P will bring-up the extension. 
Example,
.drawer-component {